File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// Ec2Shop SDK exists test

require_once __DIR__ . '/../ec2shop_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = Ec2ShopSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# Ec2Shop SDK exists test

require "minitest/autorun"
require_relative "../Ec2Shop_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = Ec2ShopSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
